bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -3536,39 +3536,39 @@ spec:
     name: csi-vxflexos
   - image: docker.io/dellemc/sdc:4.5
```

Comment:
Please update the SDC version too from 4.5 to 4.5.1.

Reply:
Addressed